app/modules/tmr.c

```diff
@@ -135,20 +141,32 @@ static int tmr_now(lua_State* L){
 	return 1; 
 }
 
-// Lua: tmr.register( id, interval, mode, function )
+static int tmr_get( lua_State *L, int stack, timer_t *tmr ) {
+  if (lua_isuserdata(L, stack)) {
```

Comment:
The code can emit a clearer error message if you query the element type first and then have `luaL_argcheck()` report a wrong type with something like "timer object or numerical ID expected".